src/blueprint/blueprint.rs

```diff
@@ -19,6 +20,7 @@ pub struct Blueprint {
   pub definitions: Vec<Definition>,
   pub schema: SchemaDefinition,
   pub server: Server,
+  pub upstream: Upstream,
```

Comment:
@aaditmshah 
While I was adding a new upstream directive and refactoring the upstream request, I encountered a conflict. During the resolution, I saw your comment about moving the upstream from the server to the blueprint. I went ahead and moved it accordingly. Could you please check to ensure everything is in order?